frontend/src/pages/IstioConfigNew/ServiceEntryForm.tsx

```diff
@@ -8,6 +8,7 @@ import { PlusCircleIcon } from '@patternfly/react-icons';
 import { style } from 'typestyle';
```

Comment:
Ignore this line. Just adding this here to capture multiple review comments in one place.

Reply:
Can you get rid of the up arrow/down arrow that increments/decrements the ports? These don't seem useful since you'd likely need to click these thousands of times to get to the desired port. They also let you enter invalid port values like negative numbers.

![Screenshot from 2023-01-25 10-35-42](https://user-images.githubusercontent.com/6226732/214607184-a96aadc1-5196-40ea-a0e1-1936f980b756.png)

Reply:
I'm not this is worth changing but the Port Number box is marked green when it is empty. Since the user never entered any value here, it seems strange to mark this with a green check mark even that it's technically correct. Not sure how Kiali handles this for other fields that are similar but the `Protocol` and the `Location` fields both have default values and are not marked as green. What do you think?

![Screenshot from 2023-01-25 10-37-23](https://user-images.githubusercontent.com/6226732/214607665-cf3963cc-8002-47ee-9133-c6ffeee90b0f.png)

Protocol field:
![Screenshot from 2023-01-25 10-37-31](https://user-images.githubusercontent.com/6226732/214608862-b0a15ea0-0d8f-430b-afbc-14fa0bf7f36b.png)

Reply:
It looks like you can have an invalid port field and still see the "preview" button. I think the preview button should only appear when the form is totally valid. Otherwise you could create invalid SE objects.

![Screenshot from 2023-01-25 10-38-03](https://user-images.githubusercontent.com/6226732/214609015-3a540b24-a9a2-4d42-984d-4104245ae02d.png)

Reply:
Can you replace the kebab with a minus button? The only option in the kebab is to "remove port" and with the kebab it takes two clicks to remove the port. Having a minus button reduces the clicks to one. The kebab also somewhat hides removing the port whereas the minus button makes it more clear how to remove a port from the form.

Reply:
> Can you get rid of the up arrow/down arrow that increments/decrements the ports? These don't seem useful since you'd likely need to click these thousands of times to get to the desired port. They also let you enter invalid port values like negative numbers.
> 
> ![Screenshot from 2023-01-25 10-35-42](https://user-images.githubusercontent.com/6226732/214607184-a96aadc1-5196-40ea-a0e1-1936f980b756.png)

This is changed know

Reply:
> I'm not this is worth changing but the Port Number box is marked green when it is empty. Since the user never entered any value here, it seems strange to mark this with a green check mark even that it's technically correct. Not sure how Kiali handles this for other fields that are similar but the `Protocol` and the `Location` fields both have default values and are not marked as green. What do you think?
> 
> ![Screenshot from 2023-01-25 10-37-23](https://user-images.githubusercontent.com/6226732/214607665-cf3963cc-8002-47ee-9133-c6ffeee90b0f.png)
> 
> Protocol field: ![Screenshot from 2023-01-25 10-37-31](https://user-images.githubusercontent.com/6226732/214608862-b0a15ea0-0d8f-430b-afbc-14fa0bf7f36b.png)

Probably this one is a bit more tricky. Because this is the default behavior of the validation, and also the default behavior in the other components of the wizards. In the case of the Protocol, this is not validated as the values are predefined (That is why the green check is not shown)

Reply:
> It looks like you can have an invalid port field and still see the "preview" button. I think the preview button should only appear when the form is totally valid. Otherwise you could create invalid SE objects.
> 
> ![Screenshot from 2023-01-25 10-38-03](https://user-images.githubusercontent.com/6226732/214609015-3a540b24-a9a2-4d42-984d-4104245ae02d.png)

Well seen! I've changed.

Reply:
> Can you replace the kebab with a minus button? The only option in the kebab is to "remove port" and with the kebab it takes two clicks to remove the port. Having a minus button reduces the clicks to one. The kebab also somewhat hides removing the port whereas the minus button makes it more clear how to remove a port from the form.

Done.

Reply:
It looks great now! Just a couple small issues.

1. I suspect this bug existed before your changes but the validation lets you specify multiple ports with the same name. When you go to create them you'll get an error back from the API.
![Screenshot from 2023-01-26 14-59-51](https://user-images.githubusercontent.com/6226732/214939432-ec74a7e9-2ea5-4e42-b841-1833560b7bfb.png)

2. There's no validation for Target Port. You can enter letters there and invalid numbers and it'll still appear valid. Sometimes kiali crashes when you do this and sometimes it'll exclude the target port field entirely from the form data. The target port validation should be the same as the port number validation. As an edge case I'm not sure if you can have multiple target ports that are the same or not.
![Screenshot from 2023-01-26 15-01-51](https://user-images.githubusercontent.com/6226732/214939892-261e000f-5bb9-4128-93c0-8b0e925bfb1b.png)

Last thing, should there be cypress tests with these changes? There may already be coverage?

Reply:
Thanks, @nrfox ! Changes addressed. 

There are no changes required in testing, I haven't seen test coverage for this. I'll suggest to create a separate issue to improve test coverage for this.